cmake/modules/autopas_OpenMP.cmake

```diff
@@ -1,4 +1,5 @@
-option(AUTOPAS_OPENMP "Activates OpenMP shared memory parallelization." OFF)
+set(AUTOPAS_OPENMP_DOC "Activates OpenMP shared memory parallelization. (requires OpenMP 4.5)")
+option(AUTOPAS_OPENMP ${AUTOPAS_OPENMP_DOC} ON)
```

Comment:
THIS CHANGES DEFAULT BEHAVIOR! But AutoPas without OpenMP is kind of pointless anyway...